### PR TITLE
fix scan null string error

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
Dumpling use `[]string` to parse metadata variables, which can't handle null variable.
```log
[2020/09/09 15:46:07.651 +08:00] [INFO] [log.go:70] ["get global metadata failed"] [task=coa-sync-prod] [unit=dump] [error="sql: Scan error on column index 43, name \"SQL_Remaining_Delay\": converting NULL to string is unsupported"] [errorVerbose="err = sql: Scan error on column index 43, name \"SQL_Remaining_Delay\": converting NULL to string is unsupported
```

### What is changed and how it works?
Use `[]sql.NullString` instead of `[]string`.

### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch
 
### Release note

- fix the problem that dumpling fails to parse metadata when some variables are NULL

### CODE REVIEW

File: v4/export/metadata_test.go [hunk 1]
```diff
@@ -115,6 +115,34 @@ func (s *testMetaDataSuite) TestMysqlWithFollowersMetaData(c *C) {
...
+	logFile := "ON.000001"
+	pos := "7502"
+	gtidSet := "6ce40be3-e359-11e9-87e0-36933cb0ca5a:1-29"
+	rows := sqlmock.NewRows([]string{"File", "Position", "Binlog_Do_DB", "Binlog_Ignore_DB", "Executed_Gtid_Set"}).
+		AddRow(logFile, pos, "", "", gtidSet)
+	mock.ExpectQuery("SHOW MASTER STATUS").WillReturnRows(rows)
+	mock.ExpectQuery("SELECT @@default_master_connection").WillReturnError(fmt.Errorf("mock error"))
+	mock.ExpectQuery("SHOW SLAVE STATUS").WillReturnRows(sqlmock.NewRows([]string{"SQL_Remaining_Delay"}).AddRow(nil))
+
+	testFilePath := "/test"
```
Comment (lance6716) [inline v4/export/metadata_test.go:134]: maybe `c.MkDir` or something in /tmp

**Review by lance6716 — COMMENTED** [review]
rest LGTM

File: v4/export/metadata_test.go (hunk 1 above)
Comment (lichunzhu) [reply v4/export/metadata_test.go:134]: It won't write file to disk since we don't use `writeGlobalMetaData`. So I think it's okay.

**Comment by lance6716** [comment]
LGTM

**Review by kennytm — APPROVED** [review]

**Pull request merged**